app/controllers/groups_controller.rb

```diff
@@ -18,6 +18,31 @@ def show
     end
   end
 
+  def generate_token
+    @group = Group.where(id: params[:id])
+    @group.group_token_generate
+    render json: @group
+  end
+
+  def group_invite
+    @group = Group.find(params[:id])
+    @current_user = current_user.id
```

Comment:
Why is this needed ?

Reply:
thats from devise gives id of current user to check if it is already present or not it not present add that user id in group member table

Reply:
I mean why can't use current_user directly, why do you need another instance variable for this ?

Reply:
truee!!! I thought declaring data at first will make it more readable, I will use directly then!!